docs/user/environment-variables.rst

```diff
@@ -1,36 +1,51 @@
 Environment Variables
 =====================
 
-Read the Docs supports environment variables when building your project.
-There are two types of them:
+Read the Docs supports two types of environment variables in project builds:
 
-* default environment variables
-* user-defined environment variables
-
-Both are merged together during the build process and exposed to all the commands executed when building the documentation.
+* `Default environment variables`_
+* `User-defined environment variables`_
 
+Both are merged together during the build process and are exposed to all of the executed commands.
```

Comment:
@stsewd is this right? I thought we didn't expose the env vars in some cases like checking out the code, but I'm not 100% sure.

Reply:
Yeah, "VCS environment" does not expose environment variables. Only in the "Build environment" they are exposed.

I documented this the other day when a user hit https://github.com/readthedocs/readthedocs.org/issues/9438#issuecomment-1189288612

Reply:
So, #9438 bug aside, the VCS env is only used for the checkout step, correct?

Is this accurate then?

```suggestion
Both are merged together during the build process and are exposed to all of the executed commands. There are two exceptions for user-defined environment variables however:

* User-defined variables are not available during the checkout step of the :doc:`build process </builds>`
* User-defined variables that are not marked as public will not be available in :docs:`pull request builds </pull-requests>`
```

Reply:
That sounds correct 👍